docs/users/odoo-openmrs.md

```diff
@@ -9,19 +9,21 @@
         participant Odoo
         OpenMRS->>Ozone: Patients
         Ozone->>Odoo: Customers
+        OpenMRS->>Ozone: Visits
         OpenMRS->>Ozone: Drug orders
         OpenMRS->>Ozone: Lab test orders
         Ozone->>Odoo: Quotations
-        Ozone->>Odoo: Order lines
+        Ozone->>Odoo: Quotation items
```

Comment:
I think we should maintain the term 'Order Lines' as it is the terminology used within Odoo. 'Quotation Items' is primarily used in the context of ERPNext.

Reply:
+1
Lets wait for @mks-d to respond on this. I think he renamed `Sale Order Lines` to `Quotation` so maybe he has a preference here.

Reply:
Done, I have reverted it on @VaishSiddharth's PR branch.